apps/common/lib/elixir/features.ex

```diff
@@ -23,7 +23,7 @@ defmodule Elixir.Features do
     Version.match?(System.version(), ">= 1.16.0")
   end
 
-  def constains_end_of_expression? do
+  def contains_end_of_expression? do
```

Comment:
In the latest commit, I ran mix format and renamed the function to `contains_set_theoretic_types?`. please rebase it when you have a chance.

Reply:
I'm gonna throw some of my changes here into a housekeeping PR, which I think is more appropriate. The changes for Elixir 1.17 seem good, no need to block them with my kibitzing here.